.github/workflows/ci.yml

```diff
@@ -39,10 +46,17 @@ jobs:
       if: startsWith(matrix.ruby_version, '2.4')
       run: |
         gem update --system --no-document
+    - uses: actions/cache@v2
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-ruby${{ matrix.ruby_version }}-${{ hashFiles('**/Gemfile.lock') }}
```

Comment:
Would we also want to include `matrix.rails_version` in the cache key, since that effects which gems we use?

Reply:
Looks like you pushed c0889138fdf0389c53d79878bca5358c6474c3fc before I submitted this review! So this comment is stale, but added a related comment here: https://github.com/github/view_component/pull/460#pullrequestreview-478832486